data/json/npcs/cabin chemist/chemist missions.json

```diff
@@ -0,0 +1,108 @@
+      "inquire": "Do you have it?",
+      "success": "Wonderful! I'm sure this will be quite helpful.",
+      "success_lie": "Where is it?",
+      "failure": "That's unfortunate. I guess I'll have to find it myself."
+    }
+  },
+{
+    "id": "MISSION_CABIN_CHEMIST_GET_100_chem_muriatic_acid",
+    "type": "mission_definition",
+    "name": "Find Some Hydrochloric Acid",
+    "description": "Obtain 25 liters of hydrochloric acid for the chemist.",
+    "goal": "MGOAL_FIND_ITEM",
+	"item": "chem_muriatic_acid",
+	"count": 100,
+    "difficulty": 7,
+    "value": 10000,
+	"origins": [ "ORIGIN_OPENER_NPC" ],          
+    "followup": "MISSION_CABIN_CHEMIST_GET_1000_GAS",   
+	"has_generic_rewards": true,
+	"dialogue": {
+      "describe": "I need 25 liters of hydrochloric acid.",
+	  "offer": "My stock of chemicals is getting a bit low at the moment. Could you get me 25 liters of hydrochloric acid?  I use it in a lot of things, and I'd be willing to pay you however I can.",
+	  "advice": "You could find some in a warehouse.",
+      "accepted": "I'll be here when you get back. Good luck!",
+      "rejected": "Someone's got to do it.",
+      "inquire": "Do you have the hydrochloric acid?",
+      "success": "Wonderful! I'm sure this will be quite helpful.",
+      "success_lie": "Where is it?",
+      "failure": "That's unfortunate. I guess I'll have to find it myself."
+    }
+  },
+{
+    "id": "MISSION_CABIN_CHEMIST_GET_1000_GAS",
+    "type": "mission_definition",
+    "name": "Find Some Gasoline",
+    "description": "Obtain 200 liters of gasoline for the chemist.",
+    "goal": "MGOAL_FIND_ITEM",
+	"item": "gasoline",
+	"count": 1000,
+    "difficulty": 10,
+    "//": "Gasoline is absurdly valuble post-apocolypse, and they're asking for a lot of gasoline.",
+    "value": 1600000,
+	"origins": [ "ORIGIN_OPENER_NPC" ],          
+    "followup": "MISSION_CABIN_CHEMIST_SET_TRADE_ROUTE",   
+	"has_generic_rewards": true,
+	"dialogue": {
+      "describe": "I need 200 liters of gas.",
+	  "offer": "I'm in a bit of a spot. I keep a generator around here to power all my scientific equipment. It has a bit of gas in it, but I've burned through that; and I can't use most of my tools without power. This may seem like a lot, but if you could get me 200 liters of gas, I'll pay you back any way I can.",
+	  "advice": "One of those old labs might have one.",
+      "accepted": "I'll be here when you get back. Good luck!",
+      "rejected": "Someone's got to do it.",
+      "inquire": "Do you have it?",
+      "success": "Wonderful! Thank you so much!",
+      "success_lie": "Where is it?",
+      "failure": "That's unfortunate. I guess I'll have to find it myself."
+    }
+  },
+{
+    "id": "MISSION_CABIN_CHEMIST_SET_TRADE_ROUTE",
+    "type": "mission_definition",
+    "name": "Establish A Trade Route",
+    "description": "Establish trade relations between the chemist and a refugee center.",
+    "goal": "MGOAL_FIND_ITEM",
+	"item": "trade_writ",

```

Comment:
If the goal is to find some item, the description should somehow mention it.

Reply:
> If the goal is to find some item, the description should somehow mention it.

That has been fixed.